File: SofiForce.Models/Models/EntityModels/ItemStoreModel.cs
using System;

namespace Models
{
    public class ItemStoreModel
    {
        public int ItemStoreId { get; set; }
        public int? ItemId { get; set; }
        public int? BranchId { get; set; }
        public int? StoreId { get; set; }
        public int? Quantity { get; set; }
        public int? OnHand { get; set; }
        public DateTime? ExpireDate { get; set; }
        public string BatchNo { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/ItemStoreModel.ts
import { ItemModel } from "./itemModel";
import { BranchModel } from "./branchModel";
import { StoreModel } from "./storeModel";

export interface ItemStoreModel {
    itemStoreId: number;
    itemId: number | null;
    branchId: number | null;
    storeId: number | null;
    quantity: number | null;
    onHand: number | null;
    expireDate: Date | null;
    batchNo: string;

}